### PR TITLE
Initial import for java-bigtable

### PR DESCRIPTION
**Pull request by kolea2**

### CODE REVIEW

File: .kokoro/build.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+cd ${scriptDir}/..
+
+# Print out Java version
+java -version
+echo ${JOB_TYPE}
+
+mvn install -B -V \
+  -DskipTests=true \
+  -Dmaven.javadoc.skip=true \
+  -Dgcloud.download.skip=true \
```
Comment (igorbernstein2) [inline .kokoro/build.sh]: Won't the integration tests fail w/o the emulator?

File: .kokoro/build.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    ;;
+javadoc)
+    mvn javadoc:javadoc javadoc:test-javadoc
+    ;;
+integration)
+    mvn -B ${INTEGRATION_TEST_ARGS} -DtrimStackTrace=false -fae verify
+    ;;
+*)
+    ;;
+esac
```
Comment (igorbernstein2) [inline .kokoro/build.sh]: newline please

File: .kokoro/continuous/lint.cfg [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+env_vars: {
+  key: "JOB_TYPE"
+  value: "lint"
+}
```
Comment (igorbernstein2) [inline .kokoro/continuous/lint.cfg]: newline please

**Comment by kolea2** [comment]
Closing in favor of #5

**Pull request closed without merging**